oshmem/mca/spml/spml.h

```diff
@@ -314,6 +314,29 @@ typedef int (*mca_spml_base_module_send_fn_t)(void *buf,
                                               int dst,
                                               mca_spml_base_put_mode_t mode);
 
+/**
+ * The routine transfers the data asynchronously from the source PE to all PEs
+ * in the OpenSHMEM job. The routine returns immediately. The source and target
+ * buffers are reusable only after the completion of the routine. After the data
+ * is transferred to the target buffers, the counter object is updated atomically.
+ * The counter object can be read either using atomic operations such as
+ * shmem_atomic_fetch or can use point-to-point synchronization routines such as
+ * shmem_wait_until and shmem_test.
+ *
+ *  @param target      The array of remote PE addresses of the objects being written.
```

Comment:
What about this ? > A symmetric data object that is large enough to receive “size” bytes of data.